lib/structures/CommandInteraction.js

```diff
@@ -95,6 +98,14 @@ class CommandInteraction extends Interaction {
                 });
                 this.data.resolved.messages = messagemap;
             }
+            //Attachments
+            if(info.data.resolved.attachments !== undefined) {
+                const attachmentsmap = new Collection(Attachment);
+                Object.entries(info.data.resolved.attachments).forEach(([id, attachment]) => {
+                    attachmentsmap.set(id, new Attachment(attachment, this._client));
```

Comment:
Passing `this._client` to the constructor is redundant (only attachment data is needed in the `Attachment` class constructor):

```suggestion
                    attachmentsmap.set(id, new Attachment(attachment));
```